src/utils.ts

```
@@ -198,16 +199,18 @@ export function findInputFiles(req: Request, log: Logger): any {
     throw new BadRequest(msg);
   }
 
-  const msg = 'Request missing expected property: "req.files.files"';
-  log.info({loc: '[POST:REQUEST_MISFORMAT]', err: msg})
-  throw new BadRequest(msg);
+  // If we reach this point, an address has been submitted and searched for
+  // but there are no files associated with the request.
+  const msg = 'Address for specified chain not found in repository';
+  log.info({loc: '[POST:ADDRESS_NOT_FOUND]', err: msg})
+  throw new NotFound(msg);
```

It's definitely better to throw not found instead of bad request.